careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
+An engineer I is capable of taking on well-defined technical problems and managing their own work each sprint.
 
-**Technical Scope:** An Engineer I is capable of taking small well-scoped components of larger projects and features and, with the mentoring and assistance of more senior engineers, completing these tasks in a reasonable time frame. Engineers at this level work with more senior members of the Engineering team to ensure they make steady progress on tasks that are assigned to them. More senior engineers may also help when they are blocked.
+**Technical Knowledge:** They have a theoretical understanding of the core concepts and best practices of software engineering and have some practical experience implementing them. They understand the basics of writing performant, well-abstracted code, but need help with complex tasks and tasks outside their specialization.
 
-Engineers at this level will have input into the estimation and planning process, and will collaborate with more senior resources to scope out work and understand the larger risks and ramifications of the changes they are making.
+**Productivity:** They are capable of completing small, well-defined tasks on time with minimal help from other engineers. They are mindful of their technical limitations and take them into account when planning their work. They focus on reliably delivering quality work on time. They review their own code first before submitting a PR.
 
-**Communication:** They are learning how to communicate well and how to deliver feedback to peers and their manager. When given a task with unclear requirements they are learning how to ask for clarification and how to identify underlying assumptions in design and implementation and how to clarify them. They understand how their work fits into the larger picture for their team, and can work with product and other engineers to help identify conflicting requirements. They will start to participate more in the technical design process, often with guidance from more senior engineers.
+**Impact:** They take on work that helps the team meet its sprint goals. They actively participate in team norms, processes, and expectations. They focus on delivering the work assigned to them in a sprint.
```

Comment:
Similarly, I'd re-write this to reflect the outcomes, rather than the inputs. Anyone can "focus on delivering the work assigned to them", but the quality of the follow-through is part of what defines the different levels.

> They deliver well-defined tasks that contribute to the team's sprint goals. [second sentence stays the same]. They deliver the work assigned to them in sprint with [choose your modifier] consistency.